src/kirby/components/page/page.component.ts

```diff
@@ -192,13 +192,10 @@ export class PageComponent implements OnInit, OnDestroy, AfterContentInit, After
     });
   }
 
-  ngAfterContentInit(): void {
+  ngAfterContentChecked(): void {
```

Comment:
Does this have a performance impact? ⚡️

Reply:
Not as i see it. The most expensive thing we do, is creating and attaching the `IntersectionObserver`. But this is safeguarded so it only happens once. 
The rest is simple DOM traversal.

There is of course a danger, that future changes will introduce performance issues here.